pkg/apiserver/debugapi/client.go

```diff
@@ -57,55 +57,76 @@ func SendRequest(client Client, req *endpoint.Request) (*httpc.Response, error)
 	}
 }
 
-func buildRelativeURI(path string, query string) string {
-	if len(query) == 0 {
-		return path
-	}
-	return fmt.Sprintf("%s?%s", path, query)
-}
-
 type tidbImplement struct {
 	fx.In
 	Client *tidb.Client
 }
 
-func (impl *tidbImplement) Get(req *endpoint.Request) (*httpc.Response, error) {
```

Comment:
Is it possible to reduce the encapsulation (for example, remove these helper structs)?